oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/registration/InMemoryClientRegistrationRepository.java

```diff
@@ -47,7 +44,7 @@
 	 * @param registrations the client registration(s)
 	 */
 	public InMemoryClientRegistrationRepository(ClientRegistration... registrations) {
-		this(Arrays.asList(registrations));
+		this.registrations = toUnmodifiableMapByRegistrationId(Stream.of(checkEmptyRegistrations(registrations, Assert::notEmpty)));
```

Comment:
Is there a reason to change this vs keep it as is?

Reply:
registrations can be null

Reply:
I think that is fine. If it is null we should throw an exception.